Articles/Blog/2022-05-build-integration-testing.md

```diff
@@ -0,0 +1,105 @@
+ 
+These facts motivated us to hold a panel discussion in SIAM PP22 conference, inviting speakers who have made major contributions to build, integration, and testing issues in scientific computing to share their experiences and opinions. In this article, panel moderator Keita Teranishi holds a conversation with some of the panelists.
+
+<br>
+
+<img src='../../images/2022-05-blog-xsdk-dependencies.png' class='page lightbox'/>[Dependency graph for the member packages of the Extreme-Scale Software Development Kit (xSDK), derived from their Spack recipes.]
+
+**Moderator:** Please briefly introduce the projects you discussed at the panel session.
+
+**Roscoe Bartlett:** I have worked on several HPC and CSE projects over the years.  I was a major contributor to the Trilinos collection of numerical support libraries and was the former Trilinos focus lead for Software Integration.  I was the software engineering and infrastructure lead for the Consortium for Advanced Simulation of Light Water Reactors (CASL).  I was the Tools and Development Environment lead for the Sandia National Laboratories Advanced Technology Development and Mitigation (ATDM) project.  And I have been a member of the DOE ECP Interoperable Design for Exascale Application Software (IDEAS) productivity and sustainability project since 2014.  More recently I have been leading build-system and integration modernization for Trilinos and the Tribal Build, Integration, and Test (TriBITS) framework for the ECP Software Development Kit (SDK) effort.
+
+**Franck Ledoux:** I’m in charge of a research and engineering group that develops, delivers, and maintains libraries and software that are used by HPC numerical codes at CEA. One of our software products, Arcane, is middleware that provides technical services to build parallel multiphysics codes. It handles parallelism concepts, mesh data structures, I/O, and so on. A key issue with such middleware is to provide an adequate testing and validating framework. This is done with the Trollnet tool.
+
+**Dan Katz:** [Parsl](http://parsl-project.org) is a library that lets a user create parallel programs composed of Python functions and external components and execute Parsl programs on any compute resource from laptops to supercomputers. [funcX](https://funcx.org) is a federated function-as-a-service system that lets a user register functions; execute them acRoscoe a federated ecosystem of funcX endpoints that includes local, HPC, and cloud systems; and share them with collaborators.
```

Comment:
... exectue them **acRoscoe** a federated ecosystem ...

I am not sure what `acRoscoe` is but is that a typo?

Reply:
Sorry, I will fix these.